### ISSUE TITLE
Build folder not ignored by Git

### ISSUE THREAD

**Issue by pschoffer** [CorsixTH/CorsixTH#1321]
#### Describe the issue
When following the [build instruction](https://github.com/CorsixTH/CorsixTH/wiki/How-To-Compile) I ended up with the game build in CorsixTH/CorsixTH. This folder isn't in the .gitignore file though and therefore when running `git status` I get this output:
```
On branch master
Your branch is up-to-date with 'origin/master'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	CorsixTH/CorsixTH

nothing added to commit but untracked files present (use "git add" to track)

```
#### Expected Behavior
I think this build folder should be in `.gitignore` file so that it is ignored by git.

#### System Information
**CorsixTH Version:** *e.g. 0.60*

**Operating System:** *Lunux - Ubuntu*

**Theme Hospital Version:** *GOG.com*

### Additional Info
I am more than happy to create a pull request with this, I would just like to get a go ahead from you guys.

**Comment by DavidFair** [comment]
I did something similar in #1310 [here](https://github.com/CorsixTH/CorsixTH/pull/1310/files/104aa3f8792213cff9871c804f8db93eaec15a34..5f48447e10628292c2013b4ceccfaa85627fa1d1#diff-a084b794bc0759e7a6b77810e01874f2)
However @TheCycoONE is encouraging people to move away from building in the root directory (at least that was the impression I got). 

The compilation instructions for Windows correctly points people to build out of source so the Linux build instructions on the wiki need updating to follow similar conventions. 

I would do this however I'm on phone only for the next couple of days and I don't fancy the prospect of writing markdown with it.
